src/D1_model_components.py

```diff
@@ -315,54 +315,48 @@ def storage_fix(model, dict_asset, **kwargs):
 def storage_optimize(model, dict_asset, **kwargs):
     storage = solph.components.GenericStorage(
         label=dict_asset["label"],
-        existing=dict_asset["capacity"]["installedCap"]["value"],
+        existing=dict_asset["storage capacity"]["installedCap"]["value"],
```

Comment:
As you use dict_asset["storage capacity"] 3 times, you could create a variable `storage_capacity = dict_asset["storage capacity"]`, it makes the code easier to read :)

Reply:
i didn't write this piece of code, i just renamed the column value. It's the same issue for dict_value['input_power'] and dict_value['output_power']. If you want me to fix the code style in this PR for all of the assets i could do so. But I'm not sure if that makes it more readable because it's always the same structure (dict[asset][subasset] etc.) that makes it easy to read. and this way you see that each time the dict_valus is called.

Reply:
For me the point was to have shorter lines, less character to read is more readable for me. But I understand it is a personal stand. Don't change it before we have more opinion on it :)

@SabineHaas, @smartie2076 - what do you think?

Reply:
I would leave it for now.